torch/testing/__init__.py

```diff
@@ -102,6 +102,10 @@ def get_all_math_dtypes(device):
 
     return dtypes
 
+def get_all_math_dtypes_and_complex(device):
```

Comment:
how about just "get_all_complex_dtypes"?

It's trivial for the user to just concatenate the lists.

Reply:
yeah I personally think either way is fine. However, as we expand the tests for complex, I think it would be useful to have a function that returns all types including complex